lizmap/www/assets/js/dataviz/dataviz.js

```diff
@@ -295,7 +296,7 @@ var lizDataviz = function() {
         return pid;
     }
 
-    function buildPlot(id, conf){
```

Comment:
Please change also the definition of the `buildPlot` function at the end of the file to integrate this new parameter, after
`var obj = {`

Reply:
@mdouchin I have done it.